pull.go

```diff
@@ -187,13 +194,16 @@ func pullImage(ctx context.Context, store storage.Store, imageName string, optio
 	registryPath := sysregistries.RegistriesConfPath(sc)
 	searchRegistries, err := getRegistries(sc)
 	if err != nil {
+		logrus.Debugf("getRegistries Error for image [%s] [%v] [%v]", imageName, err, reflect.TypeOf(err))
```

Comment:
(FWIW you can use %#v to get a full representation of `err` without having to call `reflect `explicitly)

Reply:
Neat!  Golang learning of the day for me, thanks @mtrmac!